index.html

```diff
@@ -151,10 +151,36 @@ <h3>Skills</h3>
                 <li>Codekit</li>
                 <li>GitHub</li>
                 <li>Codepen</li>
-                <li>Gitla</li>
+                <li>Gitlab</li>
                 <li>Terminal</li>
             </ul>
         </div>
     </section>
+    <footer>
+        <div>
+            <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div>
+            <form action="https://formspree.io/f/xyyopqgn" method="post">
+                <input type="text" id="name" name="name" placeholder="Full Name" maxlength="30" required>
+                <input type="email" id="email" name="email" placeholder="Email Addres" required>
+                <textarea name="textarea" maxlength="500" placeholder="Enter text here" required></textarea>
+                <div><button id="footer-button" type="submit">Get in touch</button></div>
+            </form>
+        </div>
+        <div class="social-media">
+                <ul>
+                    <li><a><img src="/images/twitter_img.png" alt="twitter"></a></li>
+                    <li><a><img src="/images/linkedin_img.png" alt="linkedin"></a></li>
+                    <li><a><img src="/images/m_img.png" alt="m_image"></a></li>
+                    <li><a><img src="/images/github_img.png" alt="github"></a></li>
+                    <li><a><img src="/images/peace_img.png" alt="peace"></a></li>
```

Comment:
- [ ] Please check these images, as they are not being shown in the page.

Reply:
Hi @Lordkaito, Could you send a screenshot, please?. In order to understand better the issue.